src/components/Select/README.md

```diff
@@ -115,3 +115,21 @@ const MyComponent = () => {
   return <Select renderFilter={renderFilter}>/* Your options here */</Select>;
 };
 ```
+
+## CSS API
+
+| Name                                        | Description                      |
+| :------------------------------------------ | :------------------------------- |
+| `--g-select-control-text-color`             | Text color                       |
+| `--g-select-control-text-color-error`       | Text color if error              |
+| `--g-select-control-placeholder-color`      | Placeholder color                |
+| `--g-select-control-background-color`       | Background color (when disabled) |
+| `--g-select-control-background-color-hover` | Background color hovered         |
+| `--g-select-control-border-width`           | Border width                     |
+| `--g-select-control-border-radius`          | Border radius                    |
+| `--g-select-control-border-color`           | Border color                     |
+| `--g-select-control-border-color-hover`     | Border color if hovered          |
+| `--g-select-control-border-color-active`    | Border color if active           |
+| `--g-select-control-border-color-error`     | Border color if error            |
+| `--g-select-control-chevron-color`          | Color of arrow                   |
+| `--g-select-control-chevron-color-disabled` | Color of disabled arrow          |
```

Comment:
`text-color-error`, `border-color-error` and `chevron-color-disabled` are redundant. Same result can be achieved with `text-color`, `border-color` and `chevron-color` becaus error and disabled states are not the internal state, but passed from outside